### DISCUSSION TITLE
Mathematics/%E7%BA%BF%E6%80%A7%E4%BB%A3%E6%95%B0%E2%85%A1%28H%29/Rings/

### DISCUSSION THREAD

**Discussion by giscus** [HobbitQia/notebook#9] [General]
# Mathematics/%E7%BA%BF%E6%80%A7%E4%BB%A3%E6%95%B0%E2%85%A1%28H%29/Rings/

HobbitQia的笔记本

https://note.hobbitqia.cc/Mathematics/%E7%BA%BF%E6%80%A7%E4%BB%A3%E6%95%B0%E2%85%A1%28H%29/Rings/

**Comment by MTDickens** [comment]
如果希望对抽代有一个基本的了解，推荐 Michael Penn 的[课程](https://www.youtube.com/playlist?list=PL22w63XsKjqxaZ-v5N4AprggFkQXgkNoP)
其他课程，GitHub 上有一个 [Awesome Maths Learning](https://github.com/tensorush/Awesome-Maths-Learning)，资料很全，可供参考。
